### ISSUE TITLE
core/loss.py confict modules/loss.py

### ISSUE THREAD

**Issue by mathpopo** [leoxiaobin/deep-high-resolution-net.pytorch#8]
class JointsMSELoss select "mean" as criterion,but in MSELoss just can select "none", "elementwise_mean","sum","error value"

**Comment by leoxiaobin** [comment]
In torch's API, the MSELoss's reduction has the options of ‘none’ | ‘mean’ | ‘sum’. ‘none’. You can find more details at https://pytorch.org/docs/stable/nn.html#torch.nn.MSELoss.
